bloglist.md

```diff
@@ -2,4 +2,4 @@
 
 
 | First Name | Last Name | Repository | Topic | First Title | Target Group |
-|---|---|---|---|---|---|
+|   Faith-|Eke---|-Work--|Foreigners Job Search Challange---|Job---|-Foreigners--|
```

Comment:
This overwrites the separate between the header and the content. Please check the markdown syntax and try again by adding a newline underneath the cells with three dashes: "---".